migrations/sql/00000013_iou_cache.up.sql

```diff
@@ -0,0 +1,12 @@
+create table iou
```

Comment:
is there a risk that this table will become prohibitively massive over time? the cross-join involved with DetailedPRCurve requires tons of IOU calculations

Reply:
I like the idea of caching, but doing it in a table seems risky

Reply:
This is a valid concern but within a dataset-model pairing the IoU's are only computed and stored once so there is a fixed limit to the size of this table. The cross-join for detailed pr curves is still limited per datum and compared to storing a raster we are only storing a lightweight row so this should be easier on memory usage as well.

There are always going to be tradeoffs. The benefit of this solution is that subsequent evaluations (with filters or different metrics to return) will be able to reuse already computed IoUs. This will mean the first computation may take minutes but everything else will be done in seconds.

Reply:
This also separates the IoU computation from the broader metric computation which i believe was causing   smaller postgres instances to alter their normal planning.

Reply:
i also have this concern but i think its worth getting in. might want to think about mechanisms for clearing the cache if the db gets big...

Reply:
Currently, the cached IoUs are tied to the lifetime of the dataset/model.